pages/auth/redirect.vue

```diff
@@ -23,8 +32,35 @@ onMounted(async () => {
       method: method as string,
       params: { code }
     })
+
+    if (signer.value) {
+      const signature = await signMessageMemo(
+        'authorize',
+        ['read:nftbook', 'write:nftbook', 'read:nftcollection', 'write:nftcollection']
+      )
+      if (!signature) { disconnect(); return router.replace('/') }
```

Comment:
```suggestion
      if (!signature) {
        disconnect()
        router.replace('/')
        return
      }
```
Prefer consistent return value for a function, in this case we originally return void at the end

Reply:
[🥅 Handle missing signer issue](https://github.com/likecoin/nft-book-press/pull/175/commits/635e5ef095ca3c85fa782a13fec9c696aa991068)